.hadolint.yaml

```diff
@@ -0,0 +1,7 @@
+ignored:
+  # ignore apt version pinning
+  - DL3008
+  # ignore pip version pinning
+  - DL3013
```

Comment:
Next time we update, we should remove this line. We do not use `pip`

Reply:
Oh, you are totally right.

Reply:
But you already did it in https://github.com/FreshRSS/FreshRSS/commit/61af33a6f58e15d88a08765046bbaf08be7dd49e
Thanks!

Reply:
I have just fixed it :-)